esmvaltool/diag_scripts/zmnam/zmnam.py

```diff
@@ -0,0 +1,99 @@
+"""
+;;#############################################################################
+;; Zonal mean Northern Annular Mode Diagnostics
+;; Author: Federico Serva (ISAC-CNR & ISMAR-CNR, Italy)
+;; Copernicus C3S 34a lot 2 (MAGIC)
+;;#############################################################################
+;; Description
+;;    Evaluation of stratosphere-troposphere coupling
+;;    based on EOF/PC analysis of the geopotential height field
+;; Modification history
+;;    20180512-A_serv_fe: Added output netCDFs, more use of preprocessor
+;;    20180510-A_serv_fe: Routines written.
+;;#############################################################################
+"""
+
+import yaml
+import sys
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import warnings
+import iris
+import iris.plot as iplt
+import iris.quickplot as qplt
+import os
+import logging
+
+from esmvaltool.diag_scripts.shared import (group_metadata, run_diagnostic,
+                                            select_metadata, sorted_metadata)
+from esmvaltool.diag_scripts.shared.plot import quickplot
+
+logger = logging.getLogger(__name__)
+
+if not sys.warnoptions:
+    warnings.simplefilter("ignore")
```

Comment:
What warnings are you ignoring?

Reply:
This `if` was found in the original python diagnostic used as template. As far as I know there are no relevant warnings, I can remove and check again if this can be removed

Reply:
Please remove it, ignoring all warnings is not a good idea, because usually they are there for a reason. What python diagnostic did you use as template?

Reply:
I believe some earlier python template, but I don't recall now...

Reply:
I removed the lines and it works without problems.